framework/src/main/java/dev/ikm/komet/framework/utils/StringUtils.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2015 Integrated Knowledge Management (support@ikm.dev)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package dev.ikm.komet.framework.utils;
+
+/**
+ * Common string operations utility
+ */
+public class StringUtils {
```

Comment:
We really don't need this common utility. Let me explain based on the following:

- Lot's of these convenient functions exist in JavaFX `TextField.textProperty()` or `StringProperty`. 
- Built-in to Java 11 has `"".isEmpty() "`  `".isBlank()`, however the object can not be null.
- For Constants like `EMPTY_STRING` just use `""`. It is maybe less readable but it's less typing and simple.
- Not sure if we use Apache commons lang's StringUtils, but I agree that we don't want another dependency for smaller capabilities. 

The objective is to try to use what's available but if we create something as helpers we should have a write up or discussion. Like common framework/reusable capabilities. We want to lessen the code to maintain over time (b/c we'd need to test, document, communicate,  etc.)

Reply:
we don't use apache commons

Reply:
will fix...

Reply:
removed stringutils